pages/ui-components/packages.md

```diff
@@ -131,24 +131,23 @@ Using only package source would result in the following:
 @forward "uswds-helpers";
 ```
 
-If you suffer from slow compiles, it may be worth experiementing with source forwarding. 
+If you suffer from slow compiles, it may be worth experimenting with source forwarding. 
 
 ## Package Sass requirements: Load paths
 Using USWDS 3.0 and packages requires compiling your Sass with load paths. Load paths tell your Sass compiler where to look for USWDS packages. Any compiler needs to include the following load paths:
```

Comment:
```suggestion
Using USWDS 3.0 and packages requires compiling your Sass with load paths. Load paths tell your Sass compiler where to look for USWDS packages. Any compiler needs to include a load path to the `@uswds/uswds/packages` directory:
```

Reply:
Updated in b05e3d4baddb9d5940f7ee911009897b7410b6b0